pyproject.toml

```diff
@@ -1,9 +1,46 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm", "wheel"]
+requires = ["setuptools>=65.0", "setuptools-scm>=7.0"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
-write_to = "src/fmu/sumo/sim2sumo/version.py"
+
+[tool.isort]
+profile = "black"
 
 [tool.black]
 line-length = 79
+
+[project]
+name = "fmu-sumo-sim2sumo"
+requires-python = ">=3.8"
+dynamic = ["version"]
+dependencies = [
+    "sumo-wrapper-python>=1.0.3",
+    "fmu-sumo-uploader",
+    "fmu-dataio",
+    "opm<=2022.4,>=2020.10.2",
+    "ecl2df",
+    "pandas<2.0",
+    "arrow",
+]
+
+[project.optional-dependencies]
+test = ["pytest"]
+dev = ["pytest", "black", "flake8"]
+
+# [tool.setuptools]
+# package-dir = { "" = "src" }
+# include-package-data = true
+# platforms = ["any"]
+#
+# [tool.setuptools.packages.find]
+# where = ["src"]
+
+[bdist_wheel]
+universal = 1
+
+[aliases]
+test = "pytest"
+
+[project.scripts]
+sim2sumo = "fmu.sumo.sim2sumo.sim2sumo:main"
```

Comment:
think we need
```
[project.entry-points.ert]
fmu_sumo_sim2sumo_jobs = fmu.sumo.sim2sumo.hook_implementations.jobs
```

for ERT to work?

Reply:
Don't think it is required, has worked without it, but no harm having it